tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```
@@ -555,6 +573,16 @@ Services:
     - --event-size 1024 --max-batch-size 100 --preload 1000000 --transport-type 1 --uamqp-transport --sync --warmup 30
     TestNames:
       Python: ProcessEventsBatchTest
+  - Test: send-event-batch
+    Arguments:
+    - --body-size 1024 --batch-size 100 --warmup 2
```

```suggestion
    - --body-size 1024 --batch-size 100 --parallel 64
```

* We should not decrease the warmup below the default unless there is a good reason
* We will likely need `parallel > 1` to bottleneck the client CPU.  Both .NET and Python use `parallel = 64` for sending event batches.